tests/auth_tests/test_basic.py

```diff
@@ -4,7 +4,6 @@
 from django.contrib.auth import get_user_model
 from django.contrib.auth.models import AnonymousUser, User
 from django.contrib.auth.tests.custom_user import CustomUser
```

Comment:
Yes, it contains some documented stuff. https://docs.djangoproject.com/en/1.7/topics/auth/customizing/#custom-users-and-testing-fixtures I could try to move the undocumented stuff at least, but was wondering if maybe the module could be deprecated entirely.

I think `skipIfCustomUser` should at least be pretty safe to deprecate with this change.
https://code.djangoproject.com/ticket/22993